migrations/curate-qdm-1.2.3-qdm-1.3.0.toml

```diff
@@ -0,0 +1,81 @@
+new_repo = "../ocsf/qdm"
+old_repo = "../ocsf/qdm-prev"
+
+log_level = "DEBUG"
+
+# Update paths
+#   Any matching paths below will be updated in the schema patch. The "new"
+#   version of these paths will win out over the old.
+update = [
+    "?.?.description", # The community is better at updating descriptions and captions than Query
+    "*.attributes.?.description", # The community is better at updating descriptions and captions than Query
+    "*.caption", 
+    "*.constraints.?", # QDM doesn't enforce constraints
+    "*.attributes.?.requirement", # QDM doesn't enforce OCSF requirements
+    "*.attributes.observable",
+
+	"objects.registry_key", "objects.registry_key.*", "objects.registry_value.name",
+	"objects.registry_value", "objects.registry_value.*", "objects.registry_value.name",
+	"*.attributes.duration.type",
+	"*.attributes.tls.group",
+]
+
+# Preserve paths
+#   Any matching paths below will be preserved in the schema patch. In other
+#   words, the "old" version of the schema will be used.
+preserve = [
+    #"*.type", # Not so sure about this one...
+    #"?.?.attributes.?.enum.*", # Keep all enums by default
+    "*.@deprecated.*", # Don't update @deprecated messages
+
+	"*.attributes.status_detail.caption",
+	"*.attributes.duration.caption",
+]
+
+# Deprecate paths
+#   Any matching paths below will be deprecated in the schema patch. They will
+#   be preserved in the new schema, but with an @deprecated annotation.
+deprecate = [
+    #"events.email_delivery_activity.*",
```

Comment:
We can safely remove email_delivery_activity since It has been updated to EmailAcitivity in all the drivers.